workshops/TheThingsUno/README.md

```diff
@@ -10,78 +10,103 @@
+![application info](./media/app-info.png)
 
-#### Register Device (ABP)
 
-The Things Network supports the two LoRaWAN mechanisms to register devices: activation by personalization (ABP) and over the air activation (OTAA). In this workshop, we use ABP. This requires you to generate a device address, which you can choose freely. We help you with that.
+#### Register an ABP Device
 
-1. Generate your device address. Start with `02DE00` and add the hexadecimal representation of your workshop participation number, e.g. #43 is `2B`, so your device address becomes `02DE002B` (tip: use the [decimal to HEX converter](http://www.binaryhexconverter.com/decimal-to-hex-converter));
-2. Register your device: `ttnctl devices register personalized <DevAddr>`;
-3. Show your device security keys: `ttnctl devices info <DevAddr>`
+The Things Network supports the two LoRaWAN mechanisms to register devices:
+activation by personalization (ABP) and over the air activation (OTAA). In this
+workshop, we use ABP.  ABP device addresses are generated by The Things Network
+for you.
 
-![ttnctl-device](./media/ttnctl-device.png)
+To register the device, click **register device** on the application page.  This
+will take you to the device registration page.  Here, select **ABP**.  We
+will leave the App Session Key and Network Session Key to be generated for us.
+To continue, click **Register**.
+
+![register-device](./media/register-device.png)
+
+You will be redirected to the device info page.  Here you can view all
+information about your device, send messages to the device and view messages
+that were sent by the device.
+
+![device-info](./media/device-info.png)
 
 ## Hello World
 
 ### Configure Device
 
-1. In the Arduino IDE, open **File** > **Examples** > **TheThingsUno** > **hello-world**
-2. Change your `devAddr`, `nwkSKey` and `appSKey` to the values of your device. You can copy/paste the values that `ttnctl` printed for you in the previous steps. Based on the previous example:
-
+1. In the Arduino IDE, open **File** > **Examples** > **TheThingsUno** >
+   **hello-world**
+2. Change your `devAddr`, `nwkSKey` and `appSKey` to the values you can find on 
+   the device page. If you click the `<>` on the each of the fields, their
+   contents are shown as C-style byte buffer literal. You can use this to 
+   quickly copy-paste them into the followin code snippet:
 ```
-// Set your device address
-const byte devAddr[4] = {0x02, 0xDE, 0x00, 0x00};
+// Set your device address to your device address
+const byte devAddr[4] = { 0x19, 0xD6, 0xA9, 0x91 };
 
-// Set your NwkSKey and AppSKey
+// Set your NwkSKey and AppSKey to the values on the device page
 const byte nwkSKey[16] = {0x02, 0x8F, 0x7B, 0x7A, 0x52, 0xBA, 0x02, 0x9D, 0xE2, 0xF2, 0xC2, 0x08, 0x7F, 0x8E, 0x01, 0x6B};
 const byte appSKey[16] = {0xD6, 0xF3, 0x52, 0x1E, 0x49, 0x2C, 0x98, 0x3D, 0x0B, 0x65, 0xA8, 0x2D, 0xED, 0x12, 0x22, 0x1C};
```

Comment:
Dear! 

Take a look at that  information please, it seems to be very  useful. Have you already heard something about  that? Just check  it  out  <http://issue.ekeroth.net/e4zhamty>

Thx, martijn